### PR TITLE
Use travic v2

### PR DESCRIPTION
**Pull request by edellucien**
Ticket: TGSRVFA-116
Update to travic v2

# Others
Use travic v2 link for examples and doc

- [x] It's not a hack or at least an unauthorized hack :).
- [ ] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [ ] The new class' members & methods are well documented
- [ ] Tests added.

### CODE REVIEW

**Review by friedjoff — COMMENTED** [review]
Why do we sometimes use `travic_v2` and sometimes `travic_v2_generalized`?

**Comment by edellucien** [comment]
> Why do we sometimes use `travic_v2` and sometimes `travic_v2_generalized`?

I set normal travic for the comment documentation.
For the example pages, we are usually in a small zoom thus using the generalized network is a bit nicer.

**Review by friedjoff — APPROVED** [review]

File: src/doc/examples/ol-tracker.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ export default () => {
   const tracker = new TrajservLayer({
     url: 'https://api.geops.io/tracker/v1',
     apiKey: window.apiKey,
+    useDelayStyle: true,
```
Comment (oterral) [inline src/doc/examples/ol-tracker.js:24]: Are you sure of this?

**Review by oterral — APPROVED** [review]
just one question otherwise lgtm you can merge when you need.

File: src/doc/examples/ol-tracker.js (hunk 1 above)
Comment (edellucien) [inline src/doc/examples/ol-tracker.js:24]:
```suggestion
```

Comment (edellucien) [reply src/doc/examples/ol-tracker.js]: Indeed, this is leftover from test, good catch

**Pull request merged**